docs/content/en/docs/getting-started/optional/cni.md

```diff
@@ -137,6 +137,33 @@ EKS Anywhere will create the required NetworkPolicy objects for its core compone
 will not delete any of the existing NetworkPolicy objects, including the ones required
    for EKS Anywhere components (listed above). The user must delete NetworkPolicy objects as needed.
 
+### EgressMasqueradeInterfaces option for Cilium plugin
```

Comment:
plugin? do we use this term other places? if not, i would suggest using maybe use `option for the Cilium CNI` or similar.

Reply:
We use plugin here: `Policy Configuration options for Cilium plugin`